src/tests/test_blindsig.py

```diff
@@ -0,0 +1,35 @@
+"""
+Test for ECC blind signatures
+"""
+import os
+import unittest
+
+from pybitmessage.pyelliptic.eccblind import ECCBlind
+
+
+class TestBlindSig(unittest.TestCase):
+    """
+    Test case for ECC blind signature
+    """
+    def test_blind_sig(self):
+        """Test full sequence using a random certifier key and a random message"""
+        # See page 127 of the paper
+        # (1) Initialization
+        signer_obj = ECCBlind()
+        point_r = signer_obj.signer_init()
+
+        # (2) Request
+        requester_obj = ECCBlind(pubkey=signer_obj.pubkey)
+        # only 64 byte messages are planned to be used in Bitmessage
+        msg = os.urandom(64)
+        msg_blinded = requester_obj.create_signing_request(point_r, msg)
```

Comment:
Maybe `self.assertNotEqual(msg, msg_blinded)`?

Reply:
Well that will always succeed as msg is a string and msg_blinded is an openssl bignum, but I can convert it before comparison, and I can also compare blinded an unblinded signature (those should both be bignums already).

As I said before, there is still no serialisation for the wire protocol, as that has to be designed first, then there will be minor refactoring here.

Reply:
They will obviously differ. That's just a tests logic (as I understand it) to compare just in case and document it. Maybe that's unnecessary.

Serialization will be probably done in the `network` package, so it's not related to the blind signature implementation itself. The signatures seems to work well. So are you going to merge this now?

Reply:
I'll add the tests and then merge.